client/webserver/site/src/js/forms.ts

```diff
@@ -520,6 +520,7 @@ export class WalletConfigForm {
     } else el = this.textInputTmpl.cloneNode(true) as HTMLElement
     this.configElements.push([opt, el])
     const input = el.querySelector('input') as ConfigOptionInput
+    input.configOpt = opt
```

Comment:
If the `opt.key` is the only thiing we need, can we store just the key as `input.dataset.configKey` instead?